ipatests/test_webui/ui_driver.py

```diff
@@ -37,16 +37,20 @@
 
 try:
     from selenium import webdriver
-    from selenium.common.exceptions import NoSuchElementException
-    from selenium.common.exceptions import InvalidElementStateException
-    from selenium.common.exceptions import StaleElementReferenceException
-    from selenium.common.exceptions import WebDriverException
-    from selenium.common.exceptions import ElementClickInterceptedException
+    from selenium.common.exceptions import (
```

Comment:
It is a good point of replacing all those imports with one import expression
But this commit will be most likely reverted when bug is fixed in firefox, so this refactoring effort will be lost.

Reply:
Agree, fixed